exporter/awsemfexporter/datapoint.go

```diff
@@ -531,6 +534,11 @@ func (dps summaryDataPointSlice) IsStaleNaNInf(i int) (bool, pcommon.Map) {
 func createLabels(attributes pcommon.Map) map[string]string {
 	labels := make(map[string]string, attributes.Len()+1)
 	attributes.Range(func(k string, v pcommon.Value) bool {
+		// we don't want to export entity related attributes as dimensions, so we skip these
+		if strings.HasPrefix(k, AWSEntityPrefix) {
```

Comment:
nit: could be cleaner with below 
```suggestion
                 if !strings.HasPrefix(k, AWSEntityPrefix) {
		labels[k] = v.AsString()
	}
```